types/wlr_pointer_constraints_v1.c

```diff
@@ -126,6 +127,10 @@ static void pointer_constraint_commit(
 		pixman_region32_copy(&constraint->region,
 			&constraint->surface->input_region);
 	}
+
+	if (updated_region) {
+		wlr_signal_emit_safe(&constraint->events.set_region, constraint);
```

Comment:
Can we set the signal data to NULL? This would allow us to change the data to an event struct in the future (e.g. with the `pending.committed` bitfield) without breaking the API.

Reply:
Hadn't thought of that, but that makes sense -- done :+1: